packages/shared-ux/page/solution_nav/src/collapse_button.scss

```
@@ -14,24 +14,24 @@
     transition-delay: 0s !important;
   }
 
-  .kbnPageTemplate__pageSideBar:hover &,
+  .kbnWithSolutionNav:hover &,
```

The sass class references need to be updated too

```suggestion
  .kbnSolutionNav__sidebar:hover &,
```